### ISSUE TITLE
Can't interpret error message

### ISSUE THREAD

**Issue by rosestevens** [PMA-2020/ppp-web#49]
## Description

Thanks for creating this - such a useful app. I would love to be able to use this to share my surveys with my collaborators for review. I tried the web app with my own survey, and am receiving an error message which I struggle to understand. Could you kindly help me with what it may indicate? @joeflack4 @jkpr

## Steps to Reproduce

1. Using this file 
[background_final_test.xlsx](https://github.com/pmaengineering/ppp-web/files/6375774/background_final_test.xlsx)

2. Uploading on http://ppp.pma2020.org/

3. Keeping all settings as is.

## Desired/expected behavior 

Normal outcome

## Actual behavior 

Error message obtained:

![image](https://user-images.githubusercontent.com/40628486/116059490-ce28d180-a678-11eb-8b78-0ab08ac795e2.png)
![image](https://user-images.githubusercontent.com/40628486/116059628-ef89bd80-a678-11eb-8a4a-f1cd790bf36e.png)

## Additional details

Any advice would be very much appreciated! Thanks!

**Comment by jkpr** [comment]
@rosestevens Thank you for your interest, and we are happy to help! We wrote this program to work with the forms that we write (we have a certain "style"), so the PPP isn't flexible to all possible ODK forms. Having said that, as I look through your form, all I had to do to get it to work was to remove the language code, so for example `label::English (en)` was changed to `label::English` along with all other language column headers. See [background_final_test_v2.xlsx](https://github.com/pmaengineering/ppp-web/files/6377777/background_final_test_v2.xlsx)
